syntax = "proto3";

package search_leader;

message SearchLeaderResponse {
    uint32 leader_id = 1; // Leader's ID
}
